### ISSUE TITLE
Version 5.0.0 doesn't contain new sections

### ISSUE THREAD

**Issue by cowboydaniel** [ligi/SurvivalManual#310]
I've just installed the new version, but it doesn't contain all the new sections I and a couple others had added.

**Comment by ligi** [comment]
which sections are that?

**Comment by cowboydaniel** [comment]
These:

Car Repair 
Basic Mechanical Skills
Blackout Driving 
Weapons

**Comment by ligi** [comment]
Thanks - they need to be indexed - will read the sections and then index them with 5.0.1

**Comment by cowboydaniel** [comment]
Awesome! Can't wait to see those

**Comment by ligi** [comment]
LGTM - just did some minor changes - but hard to find an icon for blackout driving.
Wonder if we combine "Car repair" and "Blackout driving" to a "Car" chapter. There I can find an icon.

**Comment by cowboydaniel** [comment]
Definitely can combine Car Repair with Basic Mechanical Skills, that would make the most sense

**Comment by ligi** [comment]
don't find a good image for weapons in that set we use - the closest is:

![image](https://github.com/user-attachments/assets/b6c062b6-9f2b-4726-92b9-6140a509b9b9)

kinda funny though

**Comment by cowboydaniel** [comment]
What set do you use?

**Comment by ligi** [comment]
The vector icons inside android studio - they need to be able to be tinted.

**Comment by cowboydaniel** [comment]
Ok so I spent about an hour on it today, rebuilt the app, added the new tabs, packaged as an APK, installed it. 

The icons don't look too bad![Screenshot_20250529_162321_Survival Manual.jpg](https://github.com/user-attachments/assets/18badf48-5be6-43e1-8ff3-62158d399caa)

**Comment by MEPiK** [comment]
Yeah i still dont have these and also there is no 5.0.1. Will it ever happen?
